scripts/build_renewable_profiles.py

```diff
@@ -489,7 +489,7 @@ def create_scaling_factor(
         from _helpers import mock_snakemake
 
         os.chdir(os.path.dirname(os.path.abspath(__file__)))
-        snakemake = mock_snakemake("build_renewable_profiles", technology="solar")
+        snakemake = mock_snakemake("build_renewable_profiles", technology="hydro")
```

Comment:
Just to double-check: what has been the reason behind this change?

Reply:
Because I tested hydro locally; however I can revert it back to solar if desired

Reply:
I see, thanks for the explanation! 🙂 I think, it's better to avoid changes which we actually don't need for the purpose of a particular PR. So, if you could please revert if, it would be great.

Reply:
I have reverted it.